db/osqlsession.c

```diff
@@ -206,17 +201,14 @@ inline unsigned long long osql_sess_getrqid(osql_sess_t *sess)
  */
 int osql_sess_addclient(osql_sess_t *sess)
 {
-    Pthread_mutex_lock(&sess->clients_mtx);
 #if 0
    uuidstr_t us;
    comdb2uuidstr(sess->uuid, us);
    fprintf(stderr, "\t\tADDCLNT p_sees=%p rqid=[%llx %s] sess->completed=%llx thread=%d clients=%d p_sess->iq=%p\n",
          sess, sess->rqid, us, sess->completed, pthread_self(), sess->clients+1, sess->iq);
 #endif
```

Comment:
I realize this code is #if'd out; however, maybe consider using ATOMIC_LOAD here for the reading of sess->clients?

Reply:
32 bit variable loading is atomic so in this case that would not be needed at all.